test/unit/spec/media/track/localmediatrack.js

```diff
@@ -482,18 +482,20 @@ const { defer } = require('../../../../../lib/util');
 
       context('when called with workaroundWebKitBug1208516', () => {
         let localMediaTrack = null;
-
+        let shouldGUMReject = 0;
```

Comment:
Any reason why this needs to be a counter instead of a boolean?
If it needs to be a counter, maybe rename to something like gumRejections?

Reply:
we just need single rejection  to demonstrate this issues. multiple rejections causes other problems - which I think also need to be fixed, but I am not addressing that in this PR.

Reply:
renamed the variable now.